metals/src/main/scala/scala/meta/internal/metals/SemanticTokenCapability.scala

```diff
@@ -0,0 +1,64 @@
+package scala.meta.internal.metals
+
+import org.eclipse.lsp4j._
+import java.util
+import scala.collection.JavaConverters._
+
+class SemanticTokenCapability {
```

Comment:
This shouldn't be needed.

Reply:
I will delete the class. I was under the impression that the companion object needed a class.